ui/components/app/assets/asset-list/network-filter/network-filter.tsx

```diff
@@ -137,27 +131,29 @@ const NetworkFilter = ({ handleClose }: SortControlProps) => {
             </Text>
           </Box>
           <Box display={Display.Flex} alignItems={AlignItems.center}>
-            {chainsToShow
-              .slice(0, 5) // only show a max of 5 icons overlapping
-              .map((chain, index) => {
-                const networkImageUrl =
-                  CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[
-                    chain as keyof typeof CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP
-                  ];
-                return (
-                  <AvatarNetwork
-                    key={chainId}
-                    name="All"
-                    src={networkImageUrl ?? undefined}
-                    size={AvatarNetworkSize.Sm}
-                    // overlap the icons
-                    style={{
-                      marginLeft: index === 0 ? 0 : '-20px',
-                      zIndex: 5 - index,
-                    }}
-                  />
-                );
-              })}
+            <InfoTooltip
+              position="bottom"
+              contentText={FEATURED_NETWORK_NAMES.join(', ')}
+            />
+            {FEATURED_NETWORK_CHAIN_IDS.map((chain, index) => {
+              const networkImageUrl =
+                CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[
+                  chain as keyof typeof CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP
+                ];
+              return (
+                <AvatarNetwork
+                  key={chainId}
+                  name="All"
```

Comment:
This would require localization but if it requires a new string, I think that'd be a problem

Reply:
there's an 'all' localized

Reply:
This is a metadata thing, not displayed. I changed this from a hardcoded string to the network url here: https://github.com/MetaMask/metamask-extension/pull/29112/commits/d07fd62e4e58b98da38787c4be7b132f1df62ff4